File: python/meteroid/models/slot_plan_fee.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel
from .term_rate import TermRate


@dataclasses.dataclass
class SlotPlanFee(BaseModel):
    """Slot-based fee (e.g., per-seat pricing)"""

    rates: t.List[TermRate]

    slot_unit_name: str

    minimum_count: t.Optional[int] = None

    quota: t.Optional[int] = None

File: go/slot_plan_fee.go
// this file is @generated
package meteroid

// Slot-based fee (e.g., per-seat pricing)
type SlotPlanFee struct {
	MinimumCount *int32 `json:"minimum_count,omitempty"`

	Quota *int32 `json:"quota,omitempty"`

	Rates RequiredSlice[TermRate] `json:"rates"`

	SlotUnitName string `json:"slot_unit_name"`
}
